test/tests/functional/pbs_job_comment_on_resume.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2017 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with
+# Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software - under
+# a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestJobComment(TestFunctional):
+
+    """
+    Testing job comment is accurate
+    """
+
+    def test_job_comment_on_resume(self):
+        """
+        Testing whether job comment is accurate
+        after resuming from suspended state.
+        """
+        attr = {'resources_available.ncpus': 1}
+        self.server.create_vnodes(
```

Comment:
This step isn't really needed. you can change resources of exiting node and just use that to run a job.

Reply:
Or just have jid1 request ncpus=8 (the default node has 8cpus).

Reply:
I've seen other test cases which are requesting 8 ncpus. Though never guessed that is the default value. Thanks and I'll use the same.

Reply:
I've tried with ncpus as 8. But it's failing in my VM as the ncpus by default in it is 4 and PTL doesn't seem to set nccpus as 8 during the set-up phase. I'll modify the existing nodes ncpus to one, instead.

Reply:
Interesting.  I thought PTL reverted back to 1 vnode with 8cpus and 8gb of mem.  I just looked through the revert_to_defaults() code and only found that it deletes the vnodes.  I guess it'll revert back to the real resources of the node you are running on.